com.corifeus.onenote.yml

```diff
@@ -0,0 +1,71 @@
+app-id: com.corifeus.onenote
```

Comment:
The file should be reformatted to follow https://github.com/flathub/flathub/wiki/YAML-Style-Guide